etc/_mods/local_development.yaml

```diff
@@ -0,0 +1,37 @@
+kafka_num_brokers: 1
+server_name: localhost-alt
+base_timeout: 360
+
+radar_s3_connector:
+  kafka_num_brokers: 1
+  # Todo make heap opts configurable!!! Is now locked at 3Gb!
+  kafkaHeapOpts: "-Xms128m -Xmx256m"
+radar_output:
+  javaOpts: "-Xms128m -Xmx128m"
+radar_home:
+  s3:
+    url: http://s3.localhost-alt/login
+  dashboard:
+    url: http://grafana.localhost-alt
+  logging:
+    enabled: true # either logging or monitoring must be enabled or radar-home crashes
+    url: http://graylog.localhost-alt
+  monitoring:
+    enabled: true # either logging or monitoring must be enabled or radar-home crashes
+    url: http://grafana.localhost-alt/login
+catalog_server:
+  kafka_num_brokers: 1
+  kafka_num_replication: 1
+  kafka_num_partitions: 1
+cp_kafka:
+  heapOptions: "-Xms128M -Xmx512M"
+  configurationOverrides:
+    "offsets.topic.replication.factor": 1
+cp_zookeeper:
+  servers: 1
```

Comment:
Can we the configs that make the apps run only one replica to the base.yaml? As we try to have a minimal working config in the default configuration I think it's good to have them there.

Reply:
I would rather not do that in this PR. I would need to very carefully inspect the consequences because we change default behavior. Would be good to create a new issue for this.